app/helpers/pageflow/config_helper.rb

```diff
@@ -2,7 +2,7 @@ module Pageflow
   module ConfigHelper
     include RenderJsonHelper
 
-    def editor_config_seeds
+    def editor_config_seeds(_entry)
       render_json_partial('pageflow/config/editor_seeds')
     end
```

Comment:
If we add  `entry` as a parameter here, we should pass it the partial as a local instead of referencing the `@entry` variable there.